docs/content/administration/reverse-proxies.en-us.md

```diff
@@ -381,3 +381,9 @@ If you really need to do so, to make Gitea works with sub-path (eg: `http://exam
 1. Set `[server] ROOT_URL = http://example.com/gitea/` in your `app.ini` file.
 2. Make the reverse-proxy pass `http://example.com/gitea/foo` to `http://gitea-server:3000/foo`.
 3. Make sure the reverse-proxy not decode the URI, the request `http://example.com/gitea/a%2Fb` should be passed as `http://gitea-server:3000/a%2Fb`.
+
+## Docker / Container Registry
+
+The container registry uses a fixed sub-path `/v2` which can't be changed.
```

Comment:
```suggestion
The docker container registry uses a fixed sub-path `/v2` which can't be changed.
```

Reply:
Mentioning Docker just because that's the more common name. The correct form would be "OCI Registry" or such but then knowbody knows what's that section is about. On all other places we just use "Container Registry".

Reply:
The "docker" name is very ingrained in the community, but I could see renaming it to "OCI containers" may be more correct. Just "containers" may be ambigous as there are other types of containers as well, like LXD.